tests/test_templates.py

```diff
@@ -92,3 +94,85 @@ def test_cli(tmpdir, kind, expected):
 
     files = os.listdir(project_dir)
     assert f"{project_name}.tar.gz" in files
+
+
+@pytest.mark.skipif(
+    strtobool(os.environ.get("APPVEYOR", "False").lower()),
+    reason="This test is not supported by standard appveyor",
+)
+@pytest.mark.parametrize(
+    (
+        "diag_ticket",
+        "req_cpus",
+        "req_cpu_capabilities",
+        "req_memory",
+        "req_gpus",
+        "req_gpu_compute_capability",
+        "req_gpu_memory",
+    ),
+    [("", 1, (), "2G", 0, "", "")],
+)
+def test_processor_cli(
+    tmpdir,
+    diag_ticket,
+    req_cpus,
+    req_cpu_capabilities,
+    req_memory,
+    req_gpus,
+    req_gpu_compute_capability,
+    req_gpu_memory,
+):
+    print(json.dumps(dict(os.environ), indent=4))
+    project_name = "testeval"
+
+    file_ext = "bat" if platform.system().lower() == "windows" else "sh"
+
+    files = os.listdir(tmpdir)
+    assert len(files) == 0
+
+    out = subprocess.check_output(
+        [
+            "evalutils",
+            "init-processor",
+            project_name,
+            f"--diag-ticket={diag_ticket}",
+            f"--req-cpus={req_cpus}",
+            f"--req-cpu-capabilities={req_cpu_capabilities}",
+            f"--req-memory={req_memory}",
+            f"--req-gpus={req_gpus}",
+            f"--req-gpu-compute-capability={req_gpu_compute_capability}",
+            f"--req-gpu-memory={req_gpu_memory}",
+            "--dev",
+        ],
+        cwd=tmpdir,
+    )
+
+    files = os.listdir(tmpdir)
+    assert "testeval" in files
+    assert f"Created project {project_name}" in out.decode()
+
+    project_dir = Path(tmpdir) / project_name
+
+    out = subprocess.check_output([str(project_dir / f"build.{file_ext}")])
+
+    assert "Successfully built" in out.decode()
+    assert f"Successfully tagged {project_name}:latest" in out.decode()
+    out = subprocess.check_output([str(project_dir / f"test.{file_ext}")])
+    print(out)
+
+    # Grab the results json
+    out = out.decode().splitlines()
+    start = [i for i, ln in enumerate(out) if ln == "{"]
+    end = [i for i, ln in enumerate(out) if ln == "}"]
+    result = json.loads("\n".join(out[start[0] : (end[-1] + 1)]))
+    print(result)
+
+    # check_dict(result, expected)
```

Comment:
Remove the commented out code

Reply:
This was still a WIP. I actually want to check_dict once the output format has been decided on. I might want to comment this in the future?